script.js

```diff
@@ -1,19 +1,70 @@
-const author = document.querySelector('.author');
-const book = document.querySelector('.book');
-const add = document.querySelector('.add');
-const here = document.querySelector('.here');
-const container = document.querySelector('.container');
-
-add.addEventListener('click', ev => {
-  const library = document.createElement('div');
-  const remove = document.createElement('button');
-  remove.setAttribute('type', 'button');
-  remove.innerHTML = 'delete';
-  remove.addEventListener('click', ev => {
-    container.removeChild(library);
+class BookList {
+  constructor() {
+    this.list = [];
+  }
+
+  addBook(book) {
+    this.list.push(book);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.